backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -0,0 +1,410 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Pims.Api.Areas.Tools.Models;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Services.Admin;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Areas.Tools.Helpers
+{
+    /// <summary>
+    /// ImportPropertiesHelper class, provides a way to import properties into the datasource.
+    /// </summary>
+    public class ImportPropertiesHelper
+    {
+        #region Variables
+        private readonly IPimsAdminService _pimsAdminService;
+        private readonly ILogger _logger;
+        private readonly IList<Entity.BuildingConstructionType> _buildingConstructionTypes;
+        private readonly IList<Entity.BuildingPredominateUse> _buildingPredominateUses;
+        private readonly IList<Entity.PropertyClassification> _propertyClassifications;
+        private readonly IList<Entity.PropertyStatus> _propertyStatus;
+        private readonly IList<Entity.City> _cities;
+        private readonly IList<Entity.Agency> _agencies;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a ImportPropertiesHelper class, initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="pimsAdminService"></param>
+        /// <param name="logger"></param>
+        public ImportPropertiesHelper(IPimsAdminService pimsAdminService, ILogger logger)
+        {
+            _pimsAdminService = pimsAdminService;
+            _logger = logger;
+
+            // Preload lookup lists so that they can be references quickly.
+            _buildingConstructionTypes = _pimsAdminService.BuildingConstructionType.GetAll().ToList();
```

Comment:
Do we have any performance considerations here? it seems like we should try and initialize this class as little as possible.

Reply:
The reason they're pulled in this way is for that reason.  However multiple separate calls to this (i.e. one parcel at a time) result in a loss of performance.  Since this is more for importing, I lean towards the first.

Yes you are correct, you don't want to initialize this class over and over.  And it really shouldn't be anyways.